src/modules/parameters.json

```diff
@@ -30,7 +30,9 @@
                     "navigateTo": {
                         "action": "entity",
                         "data": {
-                            "entityId": "abc"
+                            "entityId": "abc",
+                            "entityType": "program",
```

Comment:
Why did we need changes here, if we're just moving files?

Reply:
The intents.json schema was actually duplicated in two different locations in openrpc and here. They must've differed a little bit. But I think these fields it was missing were meant to be required.

Reply:
that sounds right, IIRC. okay, thanks.